### PR TITLE
DPL Analysis: enable back some arrays.

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
Requires #3226 before it can be enabled / merged.

@pzhristov is the format in the ROOT file still this?

**Comment by ktf** [comment]
There is also an fCovariance[] for which I do not know the size and a double array which needs to be flattened at the moment (and probably also in the future).

**Comment by jgrosseo** [comment]
Thanks for merging the other one. Shall we already put in where things are fine, then we can start testing? @pzhristov can you confirm the changes made here?

**Comment by ktf** [comment]
The branch names seem to be correct, so yes, I would say you can already try this out with the current converted files. I've also added some sanity checking in #3230 which should provide better error messages when a tree is not there.

File: Framework/Core/include/Framework/AnalysisDataModel.h [hunk 1]
```diff
@@ -274,34 +274,37 @@ namespace zdc
...
+DECLARE_SOA_COLUMN(ZPATowerEnergyLR, zpaTowerEnergyLR, float[5], "fZPATowerEnergyLR");
+//DECLARE_SOA_COLUMN(fZDCTDCCorrected, fZDCTDCCorrected, float[32][4], "fZDCTDCCorrected");
 DECLARE_SOA_COLUMN(Fired, fired, uint8_t, "fFired");
 } // namespace zdc
 
-DECLARE_SOA_TABLE(Zdcs, "AOD", "ZDC", zdc::CollisionId, zdc::ZEM1Energy, zdc::ZEM2Energy, zdc::Fired);
+DECLARE_SOA_TABLE(Zdcs, "AOD", "ZDC", zdc::CollisionId,
+                  zdc::ZNCTowerEnergy, zdc::ZNATowerEnergy, zdc::ZPCTowerEnergy, zdc::ZPATowerEnergy,
+                  zdc::ZNCTowerEnergyLR, zdc::ZNATowerEnergyLR, zdc::ZPCTowerEnergyLR, zdc::ZPATowerEnergyLR,
+                  zdc::ZEM1Energy, zdc::ZEM2Energy, zdc::Fired);
```
Comment (pzhristov) [inline Framework/Core/include/Framework/AnalysisDataModel.h]: The order in the structure is different: zdc::ZEM1Energy, zdc::ZEM2Energy are before the arrays. Is this important?

Comment (ktf) [reply Framework/Core/include/Framework/AnalysisDataModel.h]: No, it should not matter.

Comment (ktf) [reply Framework/Core/include/Framework/AnalysisDataModel.h]: (that said, for sanity I will rearrange this...)

Comment (pzhristov) [reply Framework/Core/include/Framework/AnalysisDataModel.h]: Thanks, then we are ready to merge.

**Pull request merged**